Space-Runners/ablo-demo-fe
# Cu 861n7a8e2/make undo redo buttons always visible in the toolbar/mihovil kovaevi

### Description (what's changed?)

Added redo/undo buttons to the object edit toolbar as per design request. Also made the toolbar scrollable on mobile

### Testing instructions

1. Add Text and edit it. Notice how you'll have Undo/Redo buttons visible. 
2. Scroll the object edit toolbar to the right on mobile. Notice how you uncover more items, like Delete
